frontend/lang/messages/en-US.json

```
@@ -241,13 +241,14 @@
     "manage-members": "Manage Members",
     "manage-members-description": "Manage the permissions of the members in your household. {manage} allows the user to access the data-management page, and {invite} allows the user to generate invitation links for other users. Group owners cannot change their own permissions.",
     "manage": "Manage",
+    "manage-household": "Manage Household",
     "invite": "Invite",
     "looking-to-update-your-profile": "Looking to Update Your Profile?",
     "default-recipe-preferences-description": "These are the default settings when a new recipe is created in your group. These can be changed for individual recipes in the recipe settings menu.",
     "default-recipe-preferences": "Default Recipe Preferences",
     "group-preferences": "Group Preferences",
     "private-group": "Private Group",
-    "private-group-description": "Setting your group to private will default all public view options to default. This overrides any individual households or recipes public view settings.",
+    "private-group-description": "Setting your group to private will default all public view options to default. This overrides any individual households or recipes public view settings",
```

```suggestion
    "private-group-description": "Setting your group to private will default all public view options to private. This overrides any individual households or recipes public view settings",
```

Aside from the error that it defaults to default this is a bit confusing in general. Is it defaulting to private and gets overwritten by individual recipe settings. Or does it set all recipes to private and it cannot be overwritten by individual settings.

---

I changed this to:
> Setting your group to private will disable all public view options. This overrides any individual public view settings

Which I think works better. There's also the household variant:
> Setting your household to private will disable all public view options. This overrides any individual public view settings

---

Works for me 👍